sample_douyin/go.mod

```diff
@@ -0,0 +1,135 @@
+module mydouyin
+
+go 1.18
+
+replace github.com/apache/thrift => github.com/apache/thrift v0.13.0
+
+require (
+	github.com/RaymondCode/simple-demo v0.0.0-20230221080337-89b8c9434e04
+	github.com/aliyun/aliyun-oss-go-sdk v2.2.6+incompatible
+	github.com/apache/thrift v0.13.0
+	github.com/cloudwego/hertz v0.5.1
+	github.com/cloudwego/kitex v0.4.4
```

Comment:
Please try to upgrade all the dependencies to the latest version. Take [this](https://github.com/cloudwego/biz-demo/blob/main/easy_note/go.mod) as an example.

Reply:
Hello, I have solved these problems and submitted a new PR.

Reply:
Hello, I have solved these problems.